app/views/layouts/application.html.erb

```diff
@@ -1,16 +1,28 @@
 <!DOCTYPE html>
 <html>
-  <head>
-    <title>MusicSnippets</title>
-    <%= csrf_meta_tags %>
-    <%= stylesheet_link_tag    'application', media: 'all', 'data-turbolinks-track': 'reload' %>
-    <%= javascript_include_tag 'application', 'data-turbolinks-track': 'reload' %>
-  </head>
+<head>
+  <%= javascript_include_tag 'application' %>
```

Comment:
I was using single before I started at thoughtbot, but since double is the style here, I plan to convert to double. Will update as I go, and likely submit a dedicated PR at some point.